### PR TITLE
tests: net: adjusting the memory buffer for ETH

### LINKED ISSUE #42857: sam_e70b_xplained: failed to run test cases tests/net/npf and tests/net/bridge
**Describe the bug**
Failed to run test cases tests/net/npf and tests/net/bridge on sam_e70b_xplained platform.

**To Reproduce**
twister -p sam_e70b_xplained --device-testing --device-serial /dev/ttyACM0 -T tests/net/ -v
see error:
**tests/net/bridge/net.eth_bridge.ip/handler.log**
I: Device I2C_2 initialized
I: Device I2C_0 initialized
E: transfer timedout MDIO
E: transfer timedout MDIO
E: transfer timedout MDIO
E: transfer timedout MDIO
I: MAC: fc:c2:3d:0d:5d:8b
E: Failed to reserve data net buffers
E: Unable to initialize ETH queue0
W: You have 1 IPv6 net_if addresses but 4 network interfaces
W: Consider increasing CONFIG_NET_IF_MAX_IPV6_COUNT value.
W: You have 1 IPv4 net_if addresses but 4 network interfaces
W: Consider increasing CONFIG_NET_IF_MAX_IPV4_COUNT value.
I: PHY (0) Link speed 100 Mb, full duplex
**tests/net/npf/net.pkt_filter/handler.log**
I: Device I2C_2 initialized
I: Device I2C_0 initialized
E: transfer timedout MDIO
E: transfer timedout MDIO
E: transfer timedout MDIO
E: transfer timedout MDIO
I: MAC: fc:c2:3d:0d:5d:8b
E: Failed to reserve data net buffers
E: Unable to initialize ETH queue0
E: Iface 0x20400cac driver API init NULL
E: Iface 0x20400cb4 driver API init NULL
W: You have 1 IPv6 net_if addresses but 3 network interfaces
W: Consider increasing CONFIG_NET_IF_MAX_IPV6_COUNT value.
I: PHY (0) Link speed 100 Mb, full duplex

**Environment**
zephyr SDK: 13.2
commit: ccf51e3cd3ae

### PR DESCRIPTION
**Pull request by Zhaoningx**
The net cases failed to run on sam_e70b_xplained
platform because failed to reserve data net buffers,so
adjusting the memory buffer to fix this.

Fixes https://github.com/zephyrproject-rtos/zephyr/issues/42857

Signed-off-by: NingX Zhao <ningx.zhao@intel.com>

### CODE REVIEW

**Review by nandojve — CHANGES_REQUESTED** [review]
Hi @Zhaoningx ,

You need adjust your commit message on few points:
 - To fix CI compliance error you need add subsystem at commit message title
 `tests: net: Adjusting the memory buffer for ETH`
 - Please use `Fixes`, instead `Fixed`, see [Commit Guidelines](https://docs.zephyrproject.org/latest/contribute/index.html#commit-guidelines)
 - Try avoid `"` character in the commit message. It is better describe the `why` including what platform you see the issue.

**Comment by Zhaoningx** [comment]
Hi @nandojve, thanks for correcting. I have addressed the comment.

**Comment by nandojve** [comment]
Hi @Zhaoningx ,

It seems that nucleo_f429zi ETH driver broke at maintree.

CC: @erwango

**Review by npitre — DISMISSED** [review]

File: tests/net/bridge/prj.conf [hunk 1]
```diff
@@ -5,8 +5,8 @@ CONFIG_NET_ETHERNET_BRIDGE=y
 CONFIG_NET_LOG=y
```
Comment (jukkar) [inline tests/net/bridge/prj.conf:5]: Your commit message mentions `CONFIG_NET_IF_MAX_IPV6_COUNT` but you are not changing its value in conf file. Please fix the commit message.

Comment (Zhaoningx) [reply tests/net/bridge/prj.conf:5]: Hi @jukkar, thanks for correcting, I have addressed it.

**Review by jukkar — DISMISSED** [review]

**Comment by Zhaoningx** [comment]
Hi @nandojve, it seems the CI's issue, could you help to review the PR?

**Review by nandojve — DISMISSED** [review]

**Comment by mbolivar-nordic** [comment]
@Zhaoningx did you want to rebase this to see if the CI error was temporary?

**Comment by Zhaoningx** [comment]
> @Zhaoningx did you want to rebase this to see if the CI error was temporary?

sure, I‘ll do it later.

**Review by nandojve — DISMISSED** [review]

**Comment by mbolivar-nordic** [comment]
I was able to rebase this within the GitHub UI, which kept the existing review in place.

**Review by mbolivar-nordic — APPROVED** [review]

**Review by nandojve — APPROVED** [review]

**Comment by mbolivar-nordic** [comment]
@rlubos could you take a look at this bug fix?

**Review by rlubos — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Pull request merged**